core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -561,9 +561,6 @@ public void refresh(String source) throws EngineException {
         try (ReleasableLock lock = readLock.acquire()) {
             ensureOpen();
             searcherManager.maybeRefreshBlocking();
-        } catch (AlreadyClosedException e) {
```

Comment:
Do we still need to catch this and then immediately rethrow otherwise it will get eaten by the general catch block below?